airflow/operators/papermill_operator.py

```diff
@@ -16,18 +16,22 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Dict
+from typing import Dict, Optional
 
+import attr
 import papermill as pm
 
-from airflow.lineage.datasets import DataSet
+from airflow.lineage.entities import File
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 
 
-class NoteBook(DataSet):
-    type_name = "jupyter_notebook"
-    attributes = ['location', 'parameters']
+@attr.s(auto_attribs=True)
+class NoteBook(File):
+    type_hint: Optional[str] = "jupyter_notebook"
```

Comment:
Let's add description and docstring for this class in the subsequent PRs

Reply:
Sure